src/extension/options-page/DeveloperComponents/DecryptPost.tsx

```diff
@@ -43,12 +42,10 @@ export function DecryptPostDeveloperMode() {
                 {encryptedTextInput}
                 <div style={{ minHeight: 200 }}>
                     <DecryptPost
-                        disableSuccessDecryptionCache
                         alreadySelectedPreviously={[]}
                         encryptedText={encryptedText}
                         onDecrypted={post => {}}
-                        onDecryptedRaw={raw => {}}
-                        people={[]}
+                        profiles={[]}
```

Comment:
default props?

Reply:
It must be explicitly provided to prevent mistakes